src/client/automation/playback/click/index.js

```diff
@@ -40,8 +40,9 @@ export default class ClickAutomation {
         this.offsetX = clickOptions.offsetX;
         this.offsetY = clickOptions.offsetY;
 
-        this.targetElementParentNodes = [];
-        this.clickEventElement        = null;
+        this.targetElementParentNodes     = [];
+        this.clickEventElement            = null;
+        this.beforeMouseDownActiveElement = null;
```

Comment:
`beforeMouseDownActiveElement` - I can't understand meaning of this var due to bad wording. Is it `activeElementBeforeMouseDown`?